web/packages/teleport/src/Sessions/SessionList/SessionJoinBtn.tsx

```diff
@@ -84,6 +86,17 @@ export const SessionJoinBtn = ({
         showCTA={showCTA}
         closeMenu={closeMenu}
       />
+      {showModeratedCTA && (
+        <Box mx={12} my={3}>
+          <ButtonLockedFeature
+            noIcon
+            height="40px"
+            event={CtaEvent.CTA_ACTIVE_SESSIONS}
+          >
+            Join as a moderator with Teleport Enterprise
+          </ButtonLockedFeature>
+        </Box>
```

Comment:
We can avoid nesting the element:
```suggestion
        <ButtonLockedFeature
          noIcon
          mx="12px"
          my={3}
          width="auto"
          height="40px"
          event={CtaEvent.CTA_ACTIVE_SESSIONS}
        >
          Join as a moderator with Teleport Enterprise
        </ButtonLockedFeature>
```

Also, we copied the element from line 48, could we create a shared component that would only have different text?